cowprotocol/cowswap
# feat(hook-store): Add cow amm deposit hook dapp

# Summary

Add CoW AMM Deposit Hook Dapp

# To Test

1- Access /#/11155111/swap/hooks/WXDAI
2- Connect your wallet and switch to any cow amm deployed chain
3- Fill some swap order
4- In Hooks > Add Post-Hook Action, the CoW AMM Deposit should appear in the "All hooks" section
5- Fill the form data and add the post-hook
6- Check with the simulation feature if it working as expected

# Background

Optional: Give background information for changes you've made, that might be difficult to explain via comments
